src/entities/friendship_history.rs

```diff
@@ -9,6 +9,8 @@ use crate::{
     components::database::{DBConnection, DatabaseComponent},
     generate_uuid_v4,
 };
+
+#[cfg_attr(test, faux::create)]
 #[derive(Clone)]
```

Comment:
remove derive

Reply:
It's needed because of app usage, it's being cloned for the health components